core/backend/src/assets/Settings/Base.Schema.json

```diff
@@ -0,0 +1,27 @@
+{
+  "$comment": "This schema describes the meta-structure of an iTwin Settings Schema file. It is used to provide a guide for the schema files themselves.",
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "additionalProperties": false,
+  "required": [
+    "schemaPrefix",
+    "description"
```

Comment:
`description` is declared as optional in `SettingSchemaGroup`. Which one is correct?

Reply:
I'd say lets make it required.

Reply:
Localization strategy TBD by application?

Reply:
Made it required.